src/styles/main.scss

```diff
@@ -1,3 +1,57 @@
+$color-hover: #e3b2b2;
+$cell-width: 100px;
+$cell-height: 100px;
+$padding-calendar: 10px;
+$cell-gap: 1px;
+$week-days: mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6;
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  text-align: center;
+}
+
+.calendar {
+  margin: 10%;
+  display: flex;
+  flex-wrap: wrap;
+  padding: #{$padding-calendar};
+  width: (7 * $cell-width) + $padding-calendar * 2 + $cell-gap * 6;
+  gap: $cell-gap;
+
+  @for $month-length from 28 through 31 {
+    &--#{month-length}:nth-child(-n + 28) {
+      display: none;
+    }
+  }
+
+  @each $start-day, $num in $week-days {
+    &--#{$start-day} {
+      margin-left: $padding-calendar + $num * ($cell-gap + $cell-width);
+    }
+  }
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: $cell-width;
+    height: $cell-height;
+    border: 1px solid black;
+    background-color: #eee;
+
+    @for $number from 1 through 31 {
+      &:nth-child(#{$number})::before {
+        content: "#{$number}";
+      }
+    }
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: $color-hover;
+      transition-duration: 0.5s;
```

Comment:
Checklist:
9. [STYLES] - Make sure to add transition style under general selector, not the one with :hover - this way transition will work smoothly both ways.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/a4453d9e-4bf5-4d89-a897-a2d57380d128)